src/components/checkbox/_checkbox.scss

```diff
@@ -38,8 +38,7 @@
     width: rem(18px);
     margin-right: .5rem;
     background-color: $ui-01;
-    border-radius: 2px;
-    border: 2px solid $ui-05;
+    border: $checkbox-border-width solid $ui-05;
```

Comment:
1px border-width:
![image](https://user-images.githubusercontent.com/4185382/27092441-f9b8c17a-5028-11e7-9c22-b8ea64f9d08b.png)

2px border-width:
![image](https://user-images.githubusercontent.com/4185382/27092488-2d3cc0e6-5029-11e7-8f18-d11062f9454e.png)

☝️ One small thing with checkbox is that the border-width helps to center the checkmark. 
To me, checkmark looks more centered using 2px border.